### ISSUE TITLE
Issue with food and health

### ISSUE THREAD

**Issue by Fr05teee** [AllTheMods/ATM-8#574]
Whenever I open ATM 8 for a new session in my world I loose all my additional health points and a I have to eat all the food again. Is there a way to fix this?

**Comment by tunse80** [comment]
I've experienced this as well. The extra, added health is reset to the normal 10 hearts each time I load a world.

**Comment by SpardaIII** [comment]
Did you find where the problem come from ?

**Comment by tunse80** [comment]
No, unfortunately I'm not knowledgeable enough to understand stuff like this. :) But, for me at least, it's been like this since I started playing ATM8.
